manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
+
+jobs:
+- name: bosh
+  templates:
+  - {name: collectd, release: collectd}
+
+properties:
+  collectd:
+    interval: 10
+    config: |
```

Comment:
> Concourse manifest is not even composed from multiple files

Concourse is actually still run through spruce (it's the single file here merged with the terraform outputs etc.).

It should be possible to put all this config in a single file, and then add it to the list of files pulled in by spruce in the various pipeline tasks.